shopify-client/src/views/ProductCreate.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <div class="center">
+    <button v-on:click="shopDetails()" type="button" id="back-btn" class="btn btn-link">
```

Comment:
nit-picking but these should be on new lines, like:

v-on:click=""
type=""
etc..